build.sbt

```
@@ -121,6 +121,21 @@ def configureJUnit(crossProject: CrossProject) = {
     libraryDependencies +=
       "com.github.sbt" % "junit-interface" % "0.13.3" % "test"
   )
+  .nativeSettings(
+    libraryDependencies += "org.scala-native" %%% "junit-runtime" % nativeVersion,
+    addCompilerPlugin("org.scala-native" % "junit-plugin" % nativeVersion cross CrossVersion.full),
+    pomPostProcess := { node =>
+      import scala.xml._
+      import scala.xml.transform._
+      new RuleTransformer(new RewriteRule{
+        override def transform(n: Node) =
+          if (n.label == "dependency" && (n \ "artifactId").text.startsWith("junit-runtime_native0.4"))
```

Let's not bake in the version, WDYT?
```suggestion
          if (n.label == "dependency" && (n \ "artifactId").text.startsWith("junit-runtime_native"))
```